luigi2.h

```diff
@@ -2805,12 +2805,43 @@ int _UICodeMessage(UIElement *element, UIMessage message, int di, void *dp) {
 	} else if (message == UI_MSG_MOUSE_DRAG && element->window->pressedButton == 1 && code->lineCount) {
 		// TODO Double-click and triple-click dragging for word and line granularity respectively.
 
+		int lineHeight = UIMeasureStringHeight();
```

Comment:
Calls to `UIMeasureStringWidth/Height` should be done after the `UIFontActivate` line below.